docs/configuration/registration/objects.rst

```diff
@@ -4,161 +4,49 @@
-according to the related objecttype resource in the Objecttypes API. Open Forms
-supports creating objects in the Objects API, such as the `ProductAanvraag objecttype`_.
+The `Objects API`_ is a standard for generic storage of objects/records according to
+your own, process-specific, data description, such as the `ProductAanvraag objecttype`_.
 
-Open Forms can be configured to create an object (of type ``ProductAanvraag``) to
-register form submissions.
+Open Forms supports registering form submisisons in the Objects API. The
+:ref:`manual (Dutch) <manual_registration_objects_api>` describes how to configure a form
+for usage with the Objects API.
 
-What does the Open Forms administator need?
-===========================================
+What does the Open Forms administrator need?
+============================================
 
-* API resource URL of object type(s) to use for registration. Open Forms does not (yet)
-  need access to the Object Types API.
-* Access the Objects API, with write permissions for the relevant object types. Open
-  Forms creates and updates records.
-* API resource URLs of document types (informatieobjecttype) in the Catalogi API -
-  attachments are created using these document types:
+* An instance of the `Objecttypes API`_ with:
 
-    - PDF summary of submitted form data
-    - CSV export of submitted form data (optional)
-    - Attachments from uploads done by the end-user through ``file`` components.
+    - an API token to access the API from Open Forms
+    - one or more objecttypes (legacy configuration requires the API resource URL for
+      these too)
 
-* Write access to the Documenten API, some attachments/files are created there and then
-  related to the record in the Objects API.
+* An instance of the `Objects API`_ (v2.2+) with:
 
-.. warning:: For forms with payment requirements, a PATCH request is made to the
-   Objects API to update the payment status. This requires a version of the Objects API
-   newer than 2.1.1, which is unreleased at the time of writing.
+    - (API) access to the above Objecttypes API
+    - an API token to access the Objects API from Open Forms
+    - write permissions for the relevant Objecttypes - Open Forms creates and updates
+      records.
 
-Configuration
-=============
+* An instance of the Catalogi API (e.g. via Open Zaak) with:
+
+    - API credentials (client ID + secret) to read the available document types (
+      informatieobjecttypen)
+    - API resource URLs of the document type to use for the PDF summary of submitted
+      form data
+    - API resource URLs of the document type to use for the CSV of submitted form data
+      (optional)
+    - API resource URLs of the document type to use for the user uploaded attachments
+      (optional, works as fallback). The document type can also be selected on each
+      individual ``file`` component
 
-Below is an example of the contents in the ``record.data`` attribute in the
```

Comment:
Decided to remove this, there is no added benefit for functional admins since they're not typically involved with form-configuration, and it's way too technical for form designers.

The Productaanvraag objecttype is linked above, that should be sufficient :)

Reply:
Yes wasn't sure what to do with this